File: Python/Easy/Filter/Math/1281. Subtract the Product and Sum of Digits of an Integer.py
def subtractProductAndSum(n: int) -> int:
    ss = list(map(lambda x: int(x), str(n)))
    xx = 1
    for i in ss:
        xx *= i
    xxx = sum(ss)
    return xx - xxx


n = 4421
print(subtractProductAndSum(n))

File: PHP/Easy/Filters/Math/1281-Subtract_the_Product_and_Sum_of_Digits_of_an_Integer.php
<?php
// https://leetcode.com/problems/subtract-the-product-and-sum-of-digits-of-an-integer/
function subtractProductAndSum($n)
{
    $possibleSum = 0;
    $a = 1;
    foreach (str_split($n) as $c) {
        $possibleSum += $c;
        $a *= $c;
    }
    return $a - $possibleSum;

}

$n = 234;
var_dump(subtractProductAndSum($n));